Testing/Temporary/CTestCostData.txt

```diff
@@ -0,0 +1 @@
+---
```

Comment:
Maybe add the `Testing/Temporary` directory to the `.gitignore` file? Or are the files in this dir used?

Reply:
no, wrong git add from my side. Let me check

Reply:
Removed in last commit
@ErikSchierboom Thank you for your review and sorry for not reading the docs in enough detail.